### DISCUSSION TITLE
Enhancement: The black transition screen needs to come back in certain sections.

### DISCUSSION THREAD

**Discussion by Eliptical** [FunkinCrew/Funkin#1615] [Ideas]
I think the black transition cutscene loading the next song needs to come back, but only for certain sections of the game.
There's only 2 parts that without the transition, looks awkward. The parts being the transition from Eggnog to Winter Horror land and Roses to Thorns. A good example is Eggnog to Winter Horror land. After the new change, the sound effect of the lights turning off has completely gone unused. Intentional or not, it is a bit weird to hear lights turning on that were already on. 
The transition from Roses to Thorns has a similar problem, but more of a nitpick than anything. 
It looked more smooth when Roses would end the black flash would come and then go and you'd be presented the bright red cutscene, a completely new environment from Roses. Now, Roses ends and then you suddenly switch to the cutscene.
Other than those two sections, the rest of the game is fine with it.

**Comment by RagdollClash** [comment]
Additionally in the current game code, Monster also has this issue as Skid and Pump all of a sudden turn into Monster when the song switches.

**Comment by Supic-7** [comment]
This seems more like a bug (or an oversight), as in previous versions transitioning in most levels has a fade to black from top to bottom, which left the UI visible, while the transition from Eggnog to Winter Horrorland immediately turns the whole screen black, UI included, and adds the noise of the lights turning off. So it looks like the transition was purposefully changed for the monster reveal, which stopped working when they removed the black transition screen entirely. 
On top of this, the camera is off-center at the start of Winter Horrorland, being set to the right of the tree instead of in the middle of it, but upon restarting the song the camera is properly centered.
